server.js

```diff
@@ -0,0 +1,18 @@
+var net = require('net');
+var fs = require('fs');
+var writeStream = fs.createWriteStream('output.txt', { flags : 'w' });
```

Comment:
The instructions said to make a unique output file with each request. Does this do that?